examples/e2e/vsphere/resource_cloudaccount.tf

```diff
@@ -1,6 +1,7 @@
-data "spectrocloud_cloudaccount_vsphere" "account" {
-  name = var.shared_vmware_cloud_account_name
-}
+# BET-3646 : Commenting below "data part until we have support for vsphere account "
```

Comment:
@akhilesh2410, we have keep data block as is and remove below resource block as datasource for vsphere cloud account is supported and resource isn't supported.
And better to not leave any comments in tf file. 

https://github.com/spectrocloud/terraform-provider-spectrocloud/blob/3ffa5988e1a4beb41b6ece64618fd45faddbc1f5/examples/e2e/vsphere/resource_cloudaccount.tf#L8-L15

Reply:
ack

Reply:
Done changes.

Reply:
@rishi-anand shouldn't we add support for vsphere cloud account resource as part of this ticket?